### PR TITLE
Add an inclusivity guide

### PR DESCRIPTION
**Pull request by texasmichelle**
Build a functioning, sustainable community by defining specific
guidelines for project leaders.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/community/203)

### CODE REVIEW

**Comment by abhi-g** [comment]
Thank you @texasmichelle for taking the initiative in calling these out.

**Comment by chrisheecho** [comment]
Amazing work Michelle. Top notch as always! LGTM

**Comment by dsdinter** [comment]
LATM - Looks Awesome To Me :)
Thanks @texasmichelle.

File: INCLUSIVITY.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+* Presence of members from frequently marginalized groups
+* Continued engagement by long-term members
+* Sentiment within the community that ideas are heard and contributions valued
+* Accountability of leaders by members
+
+## Attribution
+
+The origins of this document are an enumeration of efforts by project cofounder
+[David Aronchick](https://github.com/aronchick). This was not a solo effort and
+included support from [Jeremy Lewi](https://github.com/jlewi),
```
Comment (ewilderj) [inline INCLUSIVITY.md]:
```suggestion
included support from [Jeremy Lewi](https://github.com/jlewi), [Michelle Casbon](https://github.com/texasmichelle),
```

File: INCLUSIVITY.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
+# Kubeflow Inclusivity
+
+This document is a guide to inclusivity for leaders involved in the Kubeflow
+project. It describes conscious efforts and behavior patterns that contribute
+to an inclusive environment where Kubeflow members feel safe to collaborate.
+
+The Kubeflow community is a global group of contributors with a wide variety of
+backgrounds. As such, leaders are expected to take purposeful steps to ensure
+sustainability of the collaborative space. This is essential for project health
+and future growth. Expectations of community leaders are as follows:
```
Comment (ewilderj) [inline INCLUSIVITY.md]:
```suggestion
and future growth. We expect all community members, and especially leaders, to practice and grow in the areas covered in this document.  
```

**Pull request merged**